automation/test-linux.sh

```diff
@@ -4,18 +4,32 @@ set -ex
 
 template_name=$1
 namespace="kubevirt"
+template_local=""
+template_option=$template_name
+
+if [ -z "$KUBE_CMD" ]
+then
+    export KUBE_CMD="oc"
+    echo $KUBE_CMD
+fi
 
 image_url=""
 #set secret_ref only for rhel OSes
 secret_ref=""
 if [[ $TARGET =~ rhel.* ]]; then
   image_url="docker://quay.io/openshift-cnv/ci-common-templates-images:${TARGET}"
   secret_ref="secretRef: common-templates-container-disk-puller"
+elif [[ $TARGET =~ fedora-test.* ]]; then
```

Comment:
can you please change the name of the test to something like:
`refresh-image-fedora-test`

Reply:
Done